stdlib/LinearAlgebra/src/generic.jl

```diff
@@ -414,7 +417,10 @@ function cross(a::AbstractVector, b::AbstractVector)
     end
     a1, a2, a3 = a
     b1, b2, b3 = b
-    [a2*b3-a3*b2, a3*b1-a1*b3, a1*b2-a2*b1]
+    collect(cross((a1, a2, a3), (b1, b2, b3)))
```

Comment:
`collect` is a pretty complex function so this might have some overhead for just 3 values.

Reply:
Thanks, will switch to `[cross(...)...]`